app/views/internal/configs/show.html.erb

```diff
@@ -323,34 +337,36 @@
   </div>
 </div>
 
-<div class="row my-3" id="siteConfig">
-  <div class="card w-100">
-    <div class="card-header" id="appConfigHeader">
-      <h2 class="d-inline">Environment variables / AppConfig (readonly)</h2>
-      <button class="btn btn-secondary float-right" type="button" data-toggle="collapse"
-        data-target="#appConfigBodyContainer" aria-expanded="true" aria-controls="appConfigBodyContainer">
-        Toggle
-      </button>
-    </div>
-    <div id="appConfigBodyContainer" class="collapse show hide" aria-labelledby="appConfigHeader">
-      <table class="table">
-        <thead>
-          <tr>
-            <th>Variable</th>
-            <th>Value</th>
-          </tr>
-        </thead>
-        <tbody>
-          <% ENVied.config.variables.each do |v| %>
-          <% if v.group.in?(ENVied.required_groups) %>
-          <tr>
-            <td><%= v.name %></td>
-            <td style="max-width: 750px;"><%= ApplicationConfig[v.name] %></td>
-          </tr>
-          <% end %>
-          <% end %>
-        </tbody>
-      </table>
+<% if current_user.has_role?(:single_resource_admin, Config) %>
```

Comment:
is this because env variables contain private info?

Reply:
Yeah. Even "read only" access to the env vars is basically the keys to unlimited destruction via API calls. I think it just makes sense to be as restrictive as possible with this.

I wasn't around when this got put in, but I was a little uneasy when I saw this. In the back of my mind I wrestled with whether it makes sense to have it for folks with read access to site config and I think it should not.

We can continue messing with it but all else equal I'd like to limit this even though read access to this area is already fairly limited.